socless/integrations.py

```diff
@@ -18,20 +18,19 @@
 from .logger import socless_log
 from .vault import fetch_from_vault
 from .utils import convert_empty_strings_to_none
+from .socless import SoclessException
```

Comment:
the `socless.py` file is eventually headed for deprecation so `SoclessException` needs to find a new home